Source: http://lxde.svn.sourceforge.net/viewvc/lxde/trunk/lxsession-edit/configure.in?r1=1812&r2=1835
Upstream: Yes
Reason: Apply build loop patch to lxlauncher and lxsession-edit for cwickert.
--- lxsession-edit/configure.in   2009/07/14 05:29:52 1812
+++ lxsession-edit/configure.in   2009/07/16 15:08:26 1835
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC

